server/controllers/stock/core.js

```diff
@@ -233,7 +240,7 @@ function stockManagementProcess(inventories) {
     inventory.S_SEC = CM * inventory.delay; // stock de securite
     inventory.S_MIN = inventory.S_SEC * 2; // stock minimum
     inventory.S_MAX = (CM * inventory.purchase_interval) + inventory.S_MIN; // stock maximum
-    inventory.S_MONTH = inventory.quantity / CM; // mois de stock
+    inventory.S_MONTH = Math.round(inventory.quantity / CM); // mois de stock
```

Comment:
... I think this should be `Math.floor()`.  I'm not sure how it is used, but we do not want to tell an administrator that they have 8 months of stock, if they only have 7.8 months of stock.  Patients could die!